### PR TITLE
Compare git services in docs page

### LINKED ISSUE #3907: Compare git services in docs pages
I have come up with following list of features to compare:
```
Issue tracker
Pull/Merge requests
Squash merging
Rebase merging
Pull/Merge request reviews
Merge conflict resolution
Restrict push and merge access to certain users
Markdown support
Issues and pull/merge requests templates
Revert specific commits or a merge request from the UI
Labels
Time tracking
Multiple assignees for issues
Related issues
Confidential issues
Comment reactions
Lock Discussion
Issue Boards
Create new branches from issues
Commit graph
Web code editor
Branch manager
Repoistory topics
Repository code search
Global code search
Issue search
Global issue search
Git LFS 2.0
Built-in Container Registry
Integrated Git-powered wiki
Static Git-powerded pages
Group Milestones
Granual user roles (Code, Issues, Wiki etc)
Cherry-picking changes
GPG Signed Commits
Reject unsigned commits
Verified Committer
Subgroups: groups within groups
Custom Git Hooks
Repository Activity page
Deploy Tokens
Repository Tokens with write rights
Self-hosted
Easy upgrade process
Built-in Container Registry
External git mirroring
AD / LDAP integration
Multiple LDAP / AD server support
LDAP user synchronization
OpenId Connect support
OAuth2.0 integration (external authorization)
Two factor authentication (2FA)
Mattermost/Slack integration
Discord integration
Built-in CI/CD
External CI/CD status display
Low resource usage (RAM/CPU)
```

Idea is to compare Gitea with Gogs, GitLab, GitHub, Bitbucket ~~, Gitorious~~

Anything needs to be added/removed?

### PR DESCRIPTION
**Pull request by lafriks**
Fixes #3907

### CODE REVIEW

File: docs/content/doc/features/comparison.en-us.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    identifier: "comparison"
+---
+
+---
+name: Comparison
+---
+
+# Gitea compared to other Git hosting options
+
+To help decide if Gitea is suited fot Your needs there is how it compares to other Git self hosted options.
```
Comment (daviian) [inline docs/content/doc/features/comparison.en-us.md]: Typo at `suited fot Your needs`. I'd also write `Your` in lower case.

File: docs/content/doc/features/comparison.en-us.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+| Web code editor |
+| Branch manager |
+| Repository topics |
+| Repository code search |
+| Global code search |
+| Issue search |
+| Global issue search |
+| Git LFS 2.0 |
+| Integrated Git-powered wiki |
+| Static Git-powerded pages |
```
Comment (daviian) [inline docs/content/doc/features/comparison.en-us.md]: Typo in `Git-powerded`

File: docs/content/doc/features/comparison.en-us.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+| Group Milestones |
+| Granual user roles (Code, Issues, Wiki etc) |
```
Comment (daviian) [inline docs/content/doc/features/comparison.en-us.md]: Guess this should be `Granular`?

**Review by daviian — CHANGES_REQUESTED** [review]

**Comment by lunny** [comment]
How about group some items? And something maybe lost I know:
* Batch issues handler
* Topics
* Database supports

**Comment by lafriks** [comment]
Finished, please review

@daviian @lunny fixed

**Comment by InExtremaRes** [comment]
I don't know if it is relevant but I think Gogs support rebase merging for PRs in develop branch

**Comment by lafriks** [comment]
@InExtremaRes you are right, fixed. Had x in wrong row, gogs does not support PR squash merging strategy

**Comment by lunny** [comment]
Gogs has Branch manager, otherwise LGTM

**Review by appleboy — APPROVED** [review]

**Review by daviian — APPROVED** [review]

**Comment by daviian** [comment]
@lafriks Maybe we should clarify that we don't regularly check the other services for changed features!?

**Comment by lafriks** [comment]
@daviian added

**Review by daviian — APPROVED** [review]

**Review by tboerger — APPROVED** [review]
I would prefer markdown formatted tables instead of plain HTML...

**Comment by lafriks** [comment]
@tboerger if I manage to get it working with markdown tables, I will submit that as new PR

**Comment by lunny** [comment]
Missed render type comparison,  Gitea supports orgmode and third-part render tool, Gogs support  Jupyter Notebook.

**Pull request merged**